osquery/utils/conversions/windows/strings.cpp

```diff
@@ -88,6 +88,9 @@ std::string bstrToString(const BSTR src) {
 LONGLONG cimDatetimeToUnixtime(const std::string& src) {
   // First init the SWbemDateTime class
   ISWbemDateTime* pCimDateTime = nullptr;
+  auto pCimDateTimeManager =
```

Comment:
Need to move this down below the error check otherwise we have the same potential segfault with the nullptr.

Reply:
Oh. Of course. 

Hopefully no failures leave that assigned.